Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -592,6 +592,7 @@ messages=Messages
 metadata=Metadata
 metadataCorrupt=The metadata file is faulty, please inform the administrator.
 metadataDelete=Delete metadata
+metadataDeleteNotAllowed=Metadata is set as'Required' oder "alwaysShowing" in the ruleset
 metadataDoesNotEndWith=not end with the value
```

Comment:
From the user's point of view, a little more context may be required, for example "The deletion of the metadata is not possible because metadata is set as 'Required' oder "alwaysShowing" in the ruleset.